### PR TITLE
I7468 fixing broken storybook

### LINKED ISSUE #7468: Storybook is broken
We see a few errors now when running `yarn storybook`:

1.  `Configuration property "mozillaUserId" is not defined`

2. `@loadable/webpack-plugin(node:37738) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, mkdir '/addons-frontend/node_modules/@storybook/core/dist/public'`

Also, https://mozilla.github.io/addons-frontend/ no longer works.

### PR DESCRIPTION
**Pull request by rebmullin**
fixes #7468

### CODE REVIEW

File: webpack-common.js [hunk 1]
```diff
@@ -146,6 +145,13 @@ export function getPlugins({ excludeOtherAppLocales = true } = {}) {
     }),
   ];
 
+  if (includeLoadablePlugin) {
+    // We need this file to be written on disk so that our server code can read
+    // it. In development mode, webpack usually serves the file from memory but
+    // that's not what we want for this file.
+    plugins.push(new LoadablePlugin({ writeToDisk: true }));
```
Comment (willdurand) [inline webpack-common.js]: I believe this plugin should be listed first, but maybe not anymore.

Comment (rebmullin) [reply webpack-common.js]: thanks @willdurand. So would `unshift` work (instead of `push`)? or are you saying this is okay how it is? I am not clear..

Comment (willdurand) [reply webpack-common.js]: check out the loadable docs in case it's written. maybe it's not a requirement anymore.

Comment (rebmullin) [reply webpack-common.js]: I don't see anything about this..

Comment (willdurand) [reply webpack-common.js]: then I suppose you're good to go

**Pull request merged**